test/common.py

```diff
@@ -501,7 +502,14 @@ def test_something_subtest2(self):
       # runs test_something(4, 5, 6)
   """
   def decorator(func):
-    func._parameterize = parameters
+    prev = getattr(func, '_parameterize', None)
```

Comment:
I like this but I'm curious if @sbc100 has any thoughts on this direction with the test harness?

Reply:
Oh wow.. this is kind of cool!  I think this will allow us to remove a few hacks we have in place because this doesn't currently work.

Reply:
Yeah I remember having to work around similar cases before, where I wanted to express feature matrix but couldn't.